routes/user.js

```diff
@@ -1,16 +1,33 @@
-const router = require('express').Router();
+const express = require('express');
+const router = express.Router();
 const userCtrl = require('../controllers/user');
+const create = require('../models/workoutlog');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
const create = require("../models/workoutlog");
```